postgres/tests/common.py

```diff
@@ -11,6 +14,18 @@
 PASSWORD = 'datadog'
 DB_NAME = 'datadog_test'
 POSTGRES_VERSION = os.environ.get('POSTGRES_VERSION', None)
+os.environ["POSTGRES_IMAGE"] = "alpine"
```

Comment:
```suggestion
POSTGRES_IMAGE = "alpine"
```
Sorry for the back and forth, but actually I think we can use `env_vars` in here https://github.com/DataDog/integrations-core/blob/master/postgres/tests/conftest.py#L39
See example here https://github.com/DataDog/integrations-core/blob/67f32048f21cbdb97c4bf011f80586c844834fb6/activemq_xml/tests/conftest.py#L35